llvm/include/llvm/IR/IRBuilder.h

```diff
@@ -1079,6 +1079,14 @@ class IRBuilderBase {
                            nullptr, Name);
   }
 
+  /// Create a call to llvm.experimental_cttz_elts
+  Value *CreateCountTrailingZeroElems(Type *ResTy, Value *Mask,
+                                      const Twine &Name = "") {
+    return CreateIntrinsic(
+        Intrinsic::experimental_cttz_elts, {ResTy, Mask->getType()},
+        {Mask, getInt1(/*ZeroIsPoison=*/true)}, nullptr, Name);
```

Comment:
Although the helper will currently only ever be called when ZeroIsPoison is true, I think the helper should be able to set a different value. Maybe ZeroIsPoison could be passed in with a default value of true instead?

Reply:
Good suggestion! Uploaded a new patch.